app/models/UserAnswers.scala

```diff
@@ -24,9 +24,15 @@ import java.time.Instant
 import scala.util.{Failure, Success, Try}
 
 final case class UserAnswers(
-  id: String,
+  appaId: String,
+  userId: String,
+  paperlessReference: Boolean,
+  emailVerification: Option[Boolean],
+  bouncedEmail: Option[Boolean],
+  decryptedSensitiveUserInformation: DecryptedSensitiveUserInformation,
```

Comment:
Again remove all references to decrypted and sensitive and user and rename to something generic.